### PR TITLE
Add masterclass overview page

### PR DESCRIPTION
**Pull request by mattandrews**
Might as well add this too.

### CODE REVIEW

**Comment by mattandrews** [comment]
screen shot 2015-02-17 at 12 29 47

**Comment by jennysivapalan** [comment]
:+1: :rocket:

**Comment by mattandrews** [comment]
Okay with you @feedmypixel?

**Comment by feedmypixel** [comment]
Awesome - I'll let you mail back Jessica with this -  "you asked and we built it" :) 
- Id add a conditional to not show the preview and eventbrite links as the wont work for Masterclasses right? 
- Not all the status flags will show but that will be silent so don't worry about that. 
- The draft functionality - is that just empty? Maybe also don't include this section for Masterclasses?

:+1: great stuff

**Comment by feedmypixel** [comment]
Also https://github.com/guardian/membership-frontend/pull/210 has been added to make the OAuth staff not just perm staff so you will need to update that here also

**Comment by mattandrews** [comment]
Good point @feedmypixel – I've added preview support for Masterclasses with help from @jennysivapalan.

The EventBrite links work fine though, you just need to have access to that account.

**Comment by mattandrews** [comment]
Oh, and it shows drafts too!

**Comment by feedmypixel** [comment]
:8ball:

File: frontend/app/views/staff/eventOverview.scala.html [hunk 1]
```diff
@@ -1,4 +1,8 @@
-@(liveEvents: Seq[model.RichEvent.RichEvent], draftEvents: Seq[model.RichEvent.RichEvent])
+@(
+    liveEvents: Seq[model.RichEvent.RichEvent],
+    draftEvents: Seq[model.RichEvent.RichEvent],
+    previewType: String = "event"
```
Comment (feedmypixel) [inline frontend/app/views/staff/eventOverview.scala.html]: can we work this previewType out from doing a match on the event type?

**Pull request merged**